content/documentation/device-examples/components/SimpleButton.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Simple Button
+weight: 200
+tags: ["Simple Button"]
+---
+### Description
+The [simplebutton](https://github.com/Pi4J/pi4j-example-components/tree/main/src/main/java/com/pi4j/components) (src/main/java/com/pi4j/components) is a template class, that you can use in your own Java-project.
```

Comment:
Dead link. Probably still needs merge request in the example project?

Reply:
yes, those links need the merge in the Example-Project to work.